src/style.css

```
@@ -1 +1,92 @@
-/* Styles go here */
+/* stylelint-disable no-duplicate-selectors */
+/* stylelint-disable max-empty-lines */
+/* stylelint-disable no-descending-specificity */
+html,
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  --categories-color: #00acdc;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1200px;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+}
+
+.navigation {
+  height: 100%;
+}
+
+.navigation__list {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  height: 100%;
+}
+
+.navigation__item {
+  display: inline-block;
+  list-style-type: none;
+  height: 100%;
+  margin: 0 10px;
+}
+
+.navigation__link {
+  text-decoration: none;
+  color: black;
```

it's better to use HEX
```suggestion
  color: #000000;
```

---

Okay, will change color on HEX format. But I don't understand why all tests is failed?

---

By the way, here's test report link:
https://Zhekich123.github.io/layout_moyo-header/report/html_report/